laravel/app/Http/Controllers/ScheduleDataController.php

```diff
@@ -542,7 +460,7 @@ private function insertOption()
                 $pref_anest1 = $_REQUEST['pref_anest1'];
             }
         }
-        
+
```

Comment:
Unfortunately, github won't let me comment on the actual line since it wasn't changed. On Line 458, it has this `if (isset($_REQUEST['pref_anest1']))` followed by `if ($_REQUEST['pref_anest1'] != 0){` with no else. Any reason these can't be combined into 1 if with an && instead of having a nested if?